requirements.txt

```diff
@@ -0,0 +1,6 @@
+tweepy
+spaCy
+folium
+geopy
+tqdm
+geograpy3
```

Comment:
You could pin requirements to the latest compatible release (`~=`)  to make the environment more reproducible.

i.e. `tweepy~=4.6.0` instead `tweepy`.